chainer/training/extensions/progress_bar.py

```diff
@@ -26,6 +26,9 @@ class ProgressBar(extension.Extension):
         out: Stream to print the bar. Standard output is used by default.
 
     """
+
+    invoke_before_training = True
```

Comment:
I'm not sure if you understand what I'm saying, so I put a screen capture here. You can see progress bar disappears after the `PrintReport` output.
![aaa](https://cloud.githubusercontent.com/assets/131072/17648341/780f9d2c-624b-11e6-9a5c-c5e226839945.gif)